statutes.md

```diff
@@ -25,6 +25,8 @@ notification to the Board.
 (4) Dismissal of a member is permissible in cases of gross misconduct by
 resolution of the Members‘ Assembly.
 
+(5) Members automatically lose their membership when they haven't paid their last membership invoice 12 months after they received it. The board can extend the timeframe on an individual case.
```

Comment:
"on a case by case basis when there is sufficient reason."

Reply:
I would suggest having a simple 2 tier system as you don't want to loose members who can be long term supporters

- Full members (with voting rights) have paid the yearly membership fee
- supporters (with no voting rights) have not paid within the year.

Still supporters members would benefit from information and network of BAS. They can relay actions of BAS in their own network. Supporter can become instantly a member if he pays the individual membership fee.

We can give a Full member another nice perk (TBD).

Reply:
I would add that before excluding somebody, make sure he received the invoice.

Reply:
What if I want to pay the next 2 or 5 years fee? 
While it maybe does make sense from a rational and economic perspective, I think it would be full in line with BAS principles.